mobly/controllers/android_device_lib/services/snippet_management_service.py

```diff
@@ -69,31 +69,35 @@ def add_snippet_client(self, name, package, config=None):
         class for supported configurations.
 
     Raises:
-      Error, if a duplicated name or package is passed in.
+      Error: if a duplicated name is passed in, or the same package has
+        already been registered under the same Android user ID.
     """
     # Should not load snippet with the same name more than once.
     if name in self._snippet_clients:
       raise Error(
           self,
-          'Name "%s" is already registered with package "%s", it cannot '
-          'be used again.' % (name, self._snippet_clients[name].client.package),
+          f'Name "{name}" is already registered with snippet'
```

Comment:
The "identifier" is an impl detail that should not be exposed to users, including in the error msg